angular_app/src/app/about-me/about-me.component.html

```diff
@@ -20,50 +20,24 @@ <h1> About me </h1>
- learn several different instruments like the Piano, Saxophone, Percussion and Cello. But what really caught my attention was 
- Music Theory. Understanding musical structure and how it connects everything together felt very mathematical - it was
- like doing calculations with my ears instead of my mind. By the time that I was finishing my highschool and musical education, 
- I already knew I really enjoyed building things and understanding how they work. At that time (possibly influenced by a 
- game called SimCity), I wanted to become an Architect.
-
- <br><br>
-
- Everything changed when I enrolled in the 2011 Portuguese Physics Olympiads and won first place. After that, 
- I started attending monthly meetings in a specialized school in Coimbra (Projecto Quark!) in preparation
- for the International Physics Olympiads. I got to learn Physics, Programming and Mathematics beyond the
- standard school curriculum and started to fall in love with those subjects. That's when I decided to pursue a 
- career in research and moved to Porto in mainland Portugal to study Physics. I ended also falling in love with
- this city (and its alarming amount of seagulls) and stayed there for
- the next 10 years, as I did a Masters in Theoretical Physics and finally a PhD in Computational Physics.
-
- <br><br>
-
-In 2022, I moved to the United Kingdom to do research in Imperial College London. This is when I met Ava, who shares the 
-same love for Mathematics and Computer Science as I do, and who managed to cultivate in me an appreciation 
-for Data Structures and Algorithms (and cats). London is the perfect place for creativity and individuality, given the 
-abundance of unique shows and events happening all the time everywhere. People here are not afraid to be 
-silly and to have a good laugh. Together with my appreciation for birds, this inspired me to travel to Belgium 
-in 2024 to participate in the European Seagull Screeching competition. There, the musical skills that I
-acquired during childhood combined with my seagull survival skills acquired from Porto culminated in winning first place, 
-setting off a barrage of media attention. Now everyone and their dog knew about me. My legacy would lie not in
-my scientific discoveries, but in my seagull immitation skills.
+ <p>In the sunlit 1990s, on the lush island of Madeira, I was born into a world of rhythm and song. My days were filled with the chatter of birds and the hum of melodies. Beatles songs booming from my father's beloved vinyls, and my sister Carolina's voice echoing through the house with the ease of someone who had been singing since she was born. My mother, though not a musician, was a scholar of the natural world. With quiet devotion, she lured a parade of birds into our home's orbit, turning the air into a living orchestra. From her, I inherited not notes, but curiosity, a yearning to understand the patterns behind things, the “why” behind the "what".
```

Comment:
This not much action here, just descriptions. I don't know why it's relevant.

Reply:
After you are born in music, what does it mean? What action did you take? What did you lean?

What connection it is to your mother, why you start talking about her?